patricia/children.go

```diff
@@ -36,6 +41,16 @@ type sparseChildList struct {
 	children tries
 }
 
+func (list *sparseChildList) total() int {
```

Comment:
@glyn Could you please move this method just before `print()` so the method order in the file is the same for `sparseChildList` and `denseChildList` please?